src/utils/constants/commandInfo.ts

```diff
@@ -0,0 +1,76 @@
+// Default have this for every built in command 
+export interface ICommandInformation {
```

Comment:
This is great. I had been thinking of adding such a thing as well - I'm sure we'll want more fields here over time.